code/game/machinery/droneDispenser.dm

```diff
@@ -144,6 +144,9 @@
 
 /obj/machinery/droneDispenser/process()
 	..()
+	if(!CONFIG_GET(flag/drone_ghost_role))
+		return
+	
```

Comment:
@kevinz000 this prevents production and I assume you can get the unspent materials out on attackby().